pkg/workload/workload_test.go

```diff
@@ -889,3 +889,66 @@ func TestAdmissionCheckStrategy(t *testing.T) {
 		})
 	}
 }
+
+func TestPropagateResourceRequests(t *testing.T) {
```

Comment:
could you add multiple resource types in addition to CPU something like memory?

Reply:
added a few variations with cpu, memory, gpu (extended resource)